aggsender/types/epoch_notifier.go

```diff
@@ -5,6 +5,8 @@ import (
 	"fmt"
 )
 
+const perCent = 100.0
```

Comment:
Consider keeping reusing this constant, instead of `maxPercent` (https://github.com/agglayer/aggkit/blob/8e28051fa30de5952a3aa6a88b7818d6e9c8ccf4/aggsender/epoch_notifier_per_block.go#L13). WDYT?